modules/ROOT/pages/_partials/commons/common-p2psync-websocket-using-passive.adoc

```diff
@@ -54,17 +54,17 @@ include::{root-partials}block_tabbed_code_example.adoc[]
 <.> Optionally, choose a port to use.
 By default the system will automatically assign a port -- to over-ride this, see: <<lbl-set-network-and-port>>
 
-<.> Optionally, choose a network-interface to use.
+<.> Optionally, choose a network interface to use.
 By default the system will listen on all network interfaces -- to over-ride this see: <<lbl-set-network-and-port>>
 
-<.> Optionally, choose to sync only changes, the default is not to enable delta-sync -- see: <<delta-sync>>.
+<.> Optionally, choose to sync only changes. The default is not to enable delta-sync -- see: <<delta-sync>>.
 
 <.> Set server security.
-TLS is always enabled out-of-the-box, so you can usually omit this line.
+TLS is always enabled instantly, so you can usually omit this line.
 But you _can_, optionally, disable TLS (*not* advisable in production) -- see: <<lbl-tls-security>>
 
 <.> Set the credentials this server will present to the client for authentication.
-Here we show the default TLS authentication, which is by anonymous self-signed certificate.
+Here we show the default TLS authentication, which is an anonymous self-signed certificate.
```

Comment:
Careful here, as "by" means "using" doesn't it?
Hmm, "an" sort of works, so probably OK